docs/docs/gatsby-folder-structure.md

```diff
@@ -0,0 +1,40 @@
+## Gatsby Folder Structure:
+
+```
+/                      // Root
+|-- /.cache            // Gatsby's cache directory (created automatically)
+|-- /plugins           // (optional) To host local plugins code - link to writing plugin docs
+|-- /public            // Gatsby's output directory (created automatically)
+|-- /src               // Your project source
+    |-- /pages         // describe pages dir
+    |-- /templates     // (optional) describe templates dir
+    |-- html.js        // (optional) to customise .cache/default-html.js
+|-- /static            // (optional) Your static files that are not imported through module system.
+|-- gatsby-config.js   // describe & link to docs
+|-- gatsby-node.js     // describe & link to docs
+|-- gatsby-ssr.js      // describe & link to docs
+|-- gatsby-browser.js  // describe & link to docs
+```
+
+- **`/.cache`** This folder is created automatiacally by Gatsby. The files inside this folder is not meant for modification.
+
+- **`/plugins`** To host local plugins that are not published as an `npm` package. More info here - https://next.gatsbyjs.org/docs/plugins/
```

Comment:
Missing proper Markdown link format
+
`next.gatsbyjs.org` is not needed anymore. `v2` is merged and therefore `gatsbyjs.org` is just fine